AWS/delete_iam_user.ipynb

```diff
@@ -0,0 +1,1022 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.        "policy_name": "policies",
</pre><p>This is wrong, you are getting policies as a list, however this lego takes str as input, not list</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
There is no definition for the type for policy_name.  The lego works:
![image](https://user-images.githubusercontent.com/1514288/232952053-153befca-a4a9-48c2-a10b-5270e7f40cf7.png)